docs/source/conceptual/visual_language_models.md

```diff
@@ -0,0 +1,224 @@
+# Vision Language Models (VLM)
+
+## What is VLM?
+
+Visual Language Model (VLM) are models that consume both visual and textual inputs to generate text.
+
+These models are trained on multimodal data, which includes both images and text.
+
+VLMs can be used for a variety of tasks, such as image captioning, visual question answering, and more.
+
+<div class="flex justify-center">
+    <pre>placeholder for architecture diagram</pre>
+</div>
+
+With VLM, you can generate text from an image. For example, you can generate a caption for an image, answer questions about an image, or generate a description of an image.
+
```

Comment:
you already mentioned below definitions above, maybe you can remove that and instead say something like:
Below are couple of common use cases for vision language models:

Reply:
removed and "Below are couple of common use cases for vision language models:" phrasing used